post/post.js

```diff
@@ -0,0 +1,23 @@
+import { submitPost, getUser, checkAuth } from '../fetch-utils.js';
+
+const form = document.getElementById('post');
+
+form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    const formData = new FormData(form);
+    // create pretty object
+    const data = {
+        title: formData.get('title'),
+        body: formData.get('body'),
+        contact: formData.get('contact'),
+        checkbox: formData.get('use-email'),
+    };
+    // check if the user selected use my email, and get their email if so
```

Comment:
Commenting on your code is super helpful.  Keep up this habit and lots of people will be very grateful!